pkg/controller/operators/olm/operator.go

```diff
@@ -252,6 +260,26 @@ func newOperatorWithConfig(ctx context.Context, config *operatorConfig) (*Operat
 			return nil, err
 		}
 
+		// Register QueueInformer for olmConfig
+		olmConfigInformer := externalversions.NewSharedInformerFactoryWithOptions(
+			op.client,
+			config.resyncPeriod(),
+		).Operators().V1().OLMConfigs().Informer()
```

Comment:
Is the intention to have the OLMConfig be immediately registered as a stable v1 API?

Reply:
Reminder: we may want to create a follow-up documentation ticket to documenting how to disable copied CSVs for https://olm.operatorframework.io/ that can be tackled after feature freeze.

Reply:
Agreed.